types/react-slick/index.d.ts

```diff
@@ -9,9 +11,13 @@ import * as React from "react";
 type ComponentConstructor<TProps> = React.ComponentClass<TProps> | React.StatelessComponent<TProps>;
 
 export interface CustomArrowProps {
+  key?: string | number;
+  "data-role"?: string;
```

Comment:
This prop doesn't have an implementation and is unchangeable. So, I think it shouldn't be here.

Reply:
Ok!

Reply:
Fixed it. Please review.